File: POO/21-02-25.py
class Compra:
    def __init__(self, tipo, precio):
        self.tipo=tipo
        self.precio = precio

    def vender (self): 
        return f'La {self. tipo} cuesta $ {self.precio}'

producto1 = Compra ("tablet", 4800)

producto2 = Compra("computadora", 30000)

print(producto1.vender())
print(producto2.vender())

File: P.O.O/1er parcial/21-02-25.js
class Compra{
    constructor(tipo,precio){
        this.tipo=tipo
        this.precio=precio
    }
    vender(){
        return `La ${this.tipo} cuesta ${this.precio}`;}
}

const producto1 = new Compra("tablet", 4800);
const producto2 = new Compra("computadora", 30000);

console.log(producto1.vender());
console.log(producto2.vender());